pages/worker.js

```diff
@@ -1,5 +1,11 @@
-
+importScripts("../js/lib/scalajs-3.0.0.js")
```

Comment:
One problem with this is that it's hard-coding the version. We'll have to either rename the generated file to not include the Vyxal version in its name, or generate this pages/worker.js file itself.

Reply:
I think not including version is a better idea.